### ISSUE TITLE
No new release since April 2018 ... is the project abandoned?

### ISSUE THREAD

**Issue by ppatierno** [Swagger2Markup/swagger2markup#413]
Hi all,
what's the plan for this project?
I fixed a bug (https://github.com/Swagger2Markup/swagger2markup/pull/410) long time ago (in June) but no new release since April 2018. I see the work around OpenAPI 3.0 support which is even labeled as "help-wanted" so I don't know if it's going to happen soon. What about a new release before it? Or we should just consider this project as abandoned and taking a look at alternatives? @RobWin any thoughts?

**Comment by RobWin** [comment]
Hi, 
at lot oft work was invested into OpenAPI 3.0 support, but it wasn't completed yet.
Help is still neded to complete Swagger2Markup 2.0.

**Comment by ppatierno** [comment]
Yeah, sure about that but I was reasoning about a new 1.3.4 or 1.4.0 version without the OpenAPI 3.0 support.
Without an ETA for the 2.0, it's not a good user experience blocking any other release with bug fixes imho.

**Comment by RobWin** [comment]
Yes, but it was the decision by the maintainer and contributor . 
We didn't have bug fixes for a long time. Therefore we decided to use the master branch to develop 2.0.
Looking back from now it was a bad decision. I hoped that 2.0 would be completed more quickly, but the it seems the contributor of Open API 3.0 support also don't have much time anymore.

**Comment by ppatierno** [comment]
If I was maintainer (and contributor), I would create a 1.3.x branch (or 1.4.x) from the commit just before the first OpenAPI 3.0 contribution and cherry-pick all the subsequent bug fixes not related to OpenAPI 3.0 on it.
In this way, you could continue to use master for 2.0 but still have the possibility to release a new 1.3.x or 1.4.x with bug fixes.

**Comment by ppatierno** [comment]
@RobWin do you think that my idea is feasible to apply compatibly with your time?

**Comment by RobWin** [comment]
Sure, but I just don't have time to do it.

**Comment by ppatierno** [comment]
It's a pity because this cool project has reached a kind of dead state :-( ... and using the master for OpenAPI 3.0 was a mistake because the people who had the commitment for it, now are not working on it for a long time and maybe they won't work anymore.

**Comment by RobWin** [comment]
Yes I agreee, but if the community is still interested in Swagger2Markup, they can join our Slack Channel, I give them permissions and they can just create this bugfix branch by themselves.
This is also how we added OpenAPI v3.0 support.

**Comment by andye2004** [comment]
@RobWin could you please explain what open issues / features needs to addressed in order to finalise OpenAPI v3 support and enable the release to go ahead?

Thanks, Andy.

**Comment by RobWin** [comment]
@andye2004 Have a look at all the stories which are tagged with "Help wanted"
